go-controller/pkg/factory/mocks/ObjectCacheInterface.go

```diff
@@ -1,9 +1,11 @@
-// Code generated by mockery v2.8.0. DO NOT EDIT.
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
Are you using an unreleased version of Mockery?

Reply:
yeah, looks like I have the latest:
trozet@fedora openvpn]$ mockery --version
04 Aug 21 10:29 EDT INF Starting mockery dry-run=false version=v0.0.0-dev
v0.0.0-dev


will try to fix it